p2p/transport/quicreuse/reuse.go

```diff
@@ -76,14 +76,19 @@ type reuse struct {
 	unicast map[string] /* IP.String() */ map[int] /* port */ *reuseConn
 	// global contains connections that are listening on 0.0.0.0 / ::
 	global map[int]*reuseConn
```

Comment:
Should this be called `globalListeners`?
```suggestion
	// globalListeners contains all listeners that are listening on 0.0.0.0 / ::
	// When dialing new connection, we always prefer to dial from a connection we're listening on.
	globalListeners map[int]*reuseConn
```